src/core_plugins/kibana/public/management/sections/indices/create_index_pattern/create_index_pattern.html

```diff
@@ -41,19 +41,9 @@
             </small>
           </div>
 
-          <div class="form-group time-and-pattern">
-            <label>
-              <input
-                data-test-subj="createIndexPatternIsTimeBasedCheckBox"
-                ng-model="controller.newIndexPattern.isTimeBased"
-                type="checkbox">
-              <span translate="KIBANA-CONTAINS_TIME_BASED_EVENTS"></span>
-            </label>
-          </div>
-
-          <div class="form-group" ng-if="controller.newIndexPattern.isTimeBased">
+          <div class="form-group">
             <label>
-              <span translate="KIBANA-TIME_FIELD_NAME"></span>
+              <span translate="KIBANA-TIME_FILTER_FIELD_NAME"></span>
```

Comment:
I'm using that per this suggestion: https://github.com/elastic/kibana/issues/10444#issuecomment-280773735.

I think "time picker" might be more appropriate but I'm not sure if users know what that is.